examples/chart/teleport-cluster/charts/teleport-operator/templates/resources.teleport.dev_githubconnectors.yaml

```diff
@@ -0,0 +1,196 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Are these templates auto-generated?

Reply:
Yes, those are generated by running `make manifests` in `operator/`, I will add a leading comment